templates/www/us/2019/devconfus-2019-eventpage.html

```diff
@@ -49,12 +49,13 @@
           <p>{{ _('The primary event themes this year are') }}:</p>
           <ul>
             <li>{{ _('Containers and orchestration') }}</li>
-            <li>{{ _('Machine Learning and Artificial Intelligence') }}</li>
+            <li>{{ _('MachinelLearning and artificial intelligence') }}</li>
             <li>{{ _('Serverless computing') }}</li>
             <li>{{ _('Middleware technologies') }}</li>
-            <li>{{ _('Operating Systems') }}</li>
-            <li>{{ _('Testing, Debugging, Tracing and Software Quality') }}</li>
-            <li>{{ _('User Experience') }}</li>
+            <li>{{ _('Operating systems') }}</li>
+            <li>{{ _('Testing, debugging, tracing, and software quality') }}</li>
+            <li>{{ _('User experience') }}</li>
+ 	    <li>{{ _('Open source commuunity management and processes') }}</li>
```

Comment:
just FYI, this topic wasn't part of CfP form, only following topics were included:
    Containers and orchestration
    Machine Learning & Artificial Intelligence
    Serverless computing
    Middleware technologies
    Operating systems
    Ensuring Software Quality
    User Experience in Open Source